src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -118,6 +118,7 @@ public class MSItems
 	public static final RegistryObject<Item> TOOTHRIPPER = REGISTER.register("toothripper", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 3, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(4)), new Item.Properties().defaultDurability(1200)));
 	public static final RegistryObject<Item> SHADOWRAZOR = REGISTER.register("shadowrazor", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 4, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(4)).add(OnHitEffect.onCrit(OnHitEffect.enemyPotionEffect(() -> new MobEffectInstance(MobEffects.POISON, 100, 0)))), new Item.Properties().defaultDurability(1500)));
 	public static final RegistryObject<Item> PRINCESS_PERIL = REGISTER.register("princess_peril", () -> new WeaponItem(new WeaponItem.Builder(MSItemTypes.ORGANIC_TIER, 5, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(4)).add(OnHitEffect.onCrit(OnHitEffect.enemyPotionEffect(() -> new MobEffectInstance(MobEffects.POISON, 100, 0)))), new Item.Properties().defaultDurability(1650)));
+	public static final RegistryObject<Item> CHEESE_KNIFE = REGISTER.register("cheese_knife", () -> new WeaponItem(new WeaponItem.Builder(Tiers.IRON, 2, -2.0F).efficiency(1.0F).set(MSItemTypes.MISC_TOOL).add(OnHitEffect.backstab(3)), new Item.Properties()));
```

Comment:
This new weapon is not mentioned in the PR or changelog. Make a new PR and remove it from this one if you're able, otherwise keep it in and update the PR title/description, as well as add it to the changelog and resolve the merge conflict to get it in the right place in the creative tab.